lineapy/plugins/pipeline_writer_factory.py

```diff
@@ -2,6 +2,7 @@
 from lineapy.plugins.airflow_pipeline_writer import AirflowPipelineWriter
 from lineapy.plugins.base_pipeline_writer import BasePipelineWriter
 from lineapy.plugins.dvc_pipeline_writer import DVCPipelineWriter
+from lineapy.plugins.kubeflow_pipeline_writer import KubeflowPipelineWriter
```

Comment:
need a test to test out the new kfp integration.

Reply:
added in test_writer.py